pkg/cmd/roachtest/tests/restore.go

```diff
@@ -885,7 +898,7 @@ func (rd *restoreDriver) initRestorePerfMetrics(
 		// compute throughput as MB / node / second.
 		du := dut.GetDiskUsage(ctx, rd.c.All())
 		throughput := float64(du-startDu) / (float64(rd.sp.hardware.nodes) * testDuration)
-		rd.t.L().Printf("Usage %d , Nodes %d , Duration %f\n; Throughput: %f mb / node / second",
+		rd.t.L().Printf("Usage %d , Nodes %d , Duration %f\n; AWSVolumeThroughput: %f mb / node / second",
```

Comment:
nit: I think your find+replace accidentally changed this

Reply:
Yes.